airlock/business_logic.py

```diff
@@ -170,7 +170,7 @@ class AuditEventType(Enum):
     REQUEST_FILE_UPDATE = "REQUEST_FILE_UPDATE"
     REQUEST_FILE_WITHDRAW = "REQUEST_FILE_WITHDRAW"
     REQUEST_FILE_APPROVE = "REQUEST_FILE_APPROVE"
-    REQUEST_FILE_REJECT = "REQUEST_FILE_REJECT"
+    REQUEST_FILE_REQUEST_CHANGES = "REQUEST_FILE_REQUEST_CHANGES"
```

Comment:
am I right in thinking this old value is written into the database & we therefore need a data migration?

```
sqlite> select * from local_db_auditlog where type = 'REQUEST_FILE_REJECT'
   ...> ;
32|output_checker_2|example-workspace|01J2KN9GGT60GSEBZT7XB0989N|output/rows.csv|{"group": "default"}|2024-07-12 22:43:00.674452|REQUEST_FILE_REJECT
54|output_checker_2|example-workspace|01J2KN9GGT60GSEBZT7XB0989N|output/rows.csv|{"group": "default"}|2024-07-17 14:04:09.558955|REQUEST_FILE_REJECT
```

Reply:
> am I right...

looks like it!